tools/rnc-prettyprint.sh

```diff
@@ -0,0 +1,7 @@
+# This assumes you have trang installed via package manager
+# which installs a script to provide a "trang" command.
+# Make sure you run this before commiting any changes to the 
+# rnc files.
+
+trang csl.rnc csl.rng
```

Comment:
I haven't yet determined how well this would work, but this is the idea for using trang to provide a consistent format for the rnc files.